cswinrt/code_writers.h

```diff
@@ -4714,6 +4714,15 @@ wrapped_type_name);
         auto base_semantics = get_type_semantics(type.Extends());
         auto derived_new = std::holds_alternative<object_type>(base_semantics) ? "" : "new ";
 
+        auto gc_pressure_amount = 0;
```

Comment:
do we want same logic in write_class?  note this is the netstandard variant of it.

Reply:
ah, rebase surprise, thx

Reply:
and now it's crashing test cases again - at least things are consistent